docs/core/tools/extensibility.md

```diff
@@ -1,63 +1,69 @@
 ---
 title: .NET Core CLI extensibility model | Microsoft Docs
-description: .NET Core CLI extensibility model 
+description: Learn how you can extend the Command-line Interface (CLI) tools.
 keywords: CLI, extensibility, custom commands, .NET Core
 author: blackdwarf
 ms.author: mairaw
-ms.date: 02/06/2017
+ms.date: 03/10/2017
 ms.topic: article
 ms.prod: .net-core
 ms.technology: dotnet-cli
 ms.devlang: dotnet
 ms.assetid: fffc3400-aeb9-4c07-9fea-83bc8dbdcbf3
 ---
 
-# .NET Core CLI extensibility model
+# .NET Core CLI tools extensibility model
 
-## Overview
-This document will cover the main ways how to extend the CLI tools and explain the scenarios that drive each of them. 
-It will the outline how to consume the tools as well as provide short notes on how to build both types of tools. 
+This document covers the different ways you can extend the .NET Core Command-line Interface (CLI) tools and explain the scenarios that drive each one of them. 
```

Comment:
I would think that is implicit in the topic, no? Why else would you extend a toolset outside of adding custom tools?